.vscode/launch.json

```diff
@@ -19,9 +19,7 @@
                 "action": "openExternally",
                 "pattern": "\\bNow listening on:\\s+(https?://\\S+)"
             },
-            "env": {
-                "ASPNETCORE_ENVIRONMENT": "Development"
-            },
+            "launchSettingsProfile": "https",
```

Comment:
Why these changes?

Reply:
"https" refers to the name of the profile in launchSettings.json that should be used when VS Code launches the app: https://github.com/davidfowl/TodoApi/blob/0ffe121d0c25bc4d156200410c949007f2626126/TodoApi/Properties/launchSettings.json#L22

The advantage in using "launchSettingsProfile" over "env" is that it results in consistent behavior between Visual Studio and VS Code. With this setting, people only have to manage their local configuration in launchSettings.json and VS Code will use it as well.

It doesn't really matter if you only use this one default environment variable but it quickly becomes a pain point when you change anything and forget about setting it in the other one.

Reply:
Lets use http by default.

Reply:
When the solution is opened in Visual Studio, it defaults to the "https" profile for me (I ran git clean to make sure it's a fresh start).

It would therefore be more consistent IMO, if VS Code used the "https" profile as well.

The "https" profile starts a HTTP & HTTPS port, so "http://" access works as well.

Reply:
Yea but on non windows machines the https story is broken. I’d like that to do http by default for vs code